jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -223,6 +223,31 @@ def test_parse_jwst_nircam_level2(self, imviz_app):
             assert isinstance(data.coords, GWCS)
             assert comp.units == 'MJy/sr'
 
+        # Test duplicate label functionality
+        with fits.open(filename) as pf:
+            imviz_app.app.data_collection.clear()
+            parse_data(imviz_app.app, pf, ext='SCI', show_in_viewer=False, data_label='TEST')
+            data = imviz_app.app.data_collection[0]
+            assert data.label.endswith('DATA]')
```

Comment:
Bothers me a little that the matching text doesn't have opening bracket, but maybe that's just me. 😅 (Comment applies to all similar occurences.)

```suggestion
            assert data.label.endswith('[DATA]')
```

Reply:
Added the missing `[`